test/fixtures/non-text-between-code.copy.html

```diff
@@ -15,7 +15,7 @@
 	<p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;">
 		<span style="font-size:11pt;font-family:'PT Mono',monospace;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">
 			<span style="border:none;display:inline-block;overflow:hidden;width:559px;height:474px;">
-				<img src="https://lh7-rt.googleusercontent.com/docsz/AD_4nXfOUdieC9bo7QjPnX1ROFNOXtJPZ9xPJAQ7qhlBzsNmw8XuSlVJi-vFeFNs9mXCoDB10pBicZwOpqO5bsEYsIPc_lcCDIsWfGVw18r6kSA9nygfvJsTB44V8E5OU80p5Ts?key=l0-uU5y2NRVMw07JkC0hlbQj" width="559" height="474" style="margin-left:0px;margin-top:0px;" />
+				<img src="https://lh7-rt.googleusercontent.com/docsz/AD_4nXda7XalUBFNgDXa56fLCOOS0TXIRXc3zOV6lt7B06R_P6fgx-6t38n2HNaXJPQr1vsdXtawGBy__QCc7KBcDdwvi1IQbyS7Vl0PBwGROXCyWu3oEjM8FuiSRkqqBe-nm6o?key=l0-uU5y2NRVMw07JkC0hlbQj" width="559" height="474" style="margin-left:0px;margin-top:0px;" />
```

Comment:
Poked around a bit and it looks like these copy/paste image URLs are unique to each session (the `?key=xyz` part is consistent!). We should probably be ignoring/normalizing these URLs in `cleanCopiedHTML()`: https://github.com/Mr0grog/google-docs-to-markdown/blob/b682a343e0d20dae3d0db7e61db46fd1a2c36a0b/scripts/download-fixtures.js#L183-L196

Same for the “export” version in `cleanExportedHtml()`: https://github.com/Mr0grog/google-docs-to-markdown/blob/b682a343e0d20dae3d0db7e61db46fd1a2c36a0b/scripts/download-fixtures.js#L198-L307

Reply:
Oh, this is key is unique to each *user* not session! I will just replace with one from an anonymous session, which should give us working, non-404 URLs in tests that are still consistent.